tests/product/base_product_case.py

```diff
@@ -94,8 +94,7 @@ class BaseProductTestCase(BaseTestCase):
     down_node_connection_string = r'(\nWarning: (\[%(host)s\] )?Name lookup failed for %(host)s)*'\
                                   r'(Timed out trying to connect to %(host)s)*'
 
-    status_down_node_string = r'(\tName lookup failed for %(host)s)*' \
-                              r'(\tTimed out trying to connect to %(host)s \(tried 1 time\))*'
+    status_down_node_string = r'(\tName lookup failed for %(host)s)*'
```

Comment:
The second portion of the error message might be used on a different OS where the message includes both parts? What was the failure here?

Reply:
The output simply didnt have the second part. I did run all three supported OS-types and none of them failed with this change.